### ISSUE TITLE
Allow ALL Partial Blocks in GOO

### ISSUE THREAD

**Issue by libbybapa** [BEEmod/BEE2-items#1206]
Is there some programming reason why some partial blocks interfere with goo and others don't?  For example, if the partial block is at the top of goo it does not interfere, but when placed vertically it does.  The angled partial blocks can be placed in any position in goo without issue.  The flat blocks always interfere.  Maybe it is impossible to simply allow partial blocks to be placed in goo in any position, but if it is possible it would definitely be advantageous.  Most notably, there have been several times where I have wanted to use the 4x1x1 block vertically in goo but the position is considered invalid.

**Comment by LautaroL20** [comment]
Fixed for next release. However, you can fix it manually if you want:

You have to get rid of the following lines, wich are located near the end of the item code:

```
        "OccupiedVoxels"
            {
            "Voxel"
                {
                "Pos" "0 0 0"
                "CollideType" "COLLIDE_PHYSICS"
                "CollideAgainst" "COLLIDE_NOTHING"
                "Surface"
                    {
                    "Normal" "0 0 1"
                    }
```

---

And the end part of the partial block code must be like this:

```
                "Name" "instances/BEE2/logic/blocks/128x32x32_black.vmf"
                "EntityCount" "0"
                "BrushCount" "1"
                "BrushSideCount" "6"
                }
            }
        "TargetName" "bl"
        "Offset" "64 64 64"
        }
    }
```

**Comment by libbybapa** [comment]
I'm not sure how to make the edit you propose.  All of the 'packages' appear to be zipped folders.  If I try to extract a folder it says 'an unexpected error...'.  If I try to open one of the files within the zipped folder, it says 'an unexpected error...'.  If I try to copy or even open one of the files from within the zipped folder, same thing.

**Comment by LautaroL20** [comment]
That would happen you are not experienced at all, its not complicated to do if you play atention at what are you editing. But if you are worried to break something, just wait for next release.

**Comment by libbybapa** [comment]
I'm not experienced and I am not worried about breaking anything.  How about giving the instructions on how to edit the files in the zipped folders?

**Comment by LautaroL20** [comment]
Ok, ill record a GIF / Video for you showing the steps.

**Comment by TheDarkBomber** [comment]
Download the Source Code.

**Comment by libbybapa** [comment]
Ok.  Cool.  I got it working.  

For clarity (and anyone who finds this thread) I downloaded the source code and extracted it to a folder.  At first I tried copying the 'Blocks' folder from inside the 'Packages folder', editing the editoritems.txt in the described manner and copying it into the BEE2 Packages location and deleting the 'Blocks.zip' folder that was already in there.  That did not work.  

I then deleted the existing 'Packages' folder that contained all the zip folders, and replaced it with the extracted 'Packages' folder from the source code zip.  After editing the file, that DID work.  

Awesome to know.  Thanks.
